subsys/bluetooth/audio/shell/audio.h

```diff
@@ -283,8 +283,12 @@ static void print_ltv_array(const struct shell *sh, size_t indent, const uint8_t
 		.cnt = 0U,
 		.indent = indent,
 	};
+	int err;
 
-	bt_audio_data_parse(ltv_data, ltv_data_len, print_ltv_elem, &ltv_info);
+	err = bt_audio_data_parse(ltv_data, ltv_data_len, print_ltv_elem, &ltv_info);
+	if (err != 0 && err != -ECANCELED) {
+		shell_print(sh, "ould not parse the data: %d", err);
```

Comment:
```suggestion
		shell_error(sh, "%*sInvalid LTV data: %d", indent, "", err);
```

Suggest to print as error. Also take the indent level into account.